src/ui/utils.js

```diff
@@ -478,3 +493,31 @@ function getTableCellAtPosition( position ) {
 
 	return isTableCellSelected ? position.nodeAfter : findAncestor( 'tableCell', position );
 }
+
+// Returns bounding rect for list of rects.
+//
+// @param {Array.<module:utils/dom/rect~Rect>|Array.<*>} list List of `Rect`s or any list to map by `mapFn`.
+// @param {Function} [mapFn] Mapping function for list elements.
+// @returns {module:utils/dom/rect~Rect}
+function createBoundingRect( list, mapFn = null ) {
```

Comment:
The `null` default value isn't tested (coverage dropped):

![](https://user-images.githubusercontent.com/247363/77895163-a9e0a280-7276-11ea-8467-172d0ef58dcc.png)

The other branch is below.